scripts/tippecanoe/1.22.2/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: generic
+
+sudo: false
```

Comment:
this should be nested inside the `matrix:include` items. Otherwise it will trigger an odd travis behavior that results in multiple builds.

Also `tippecanoe` does not technically require `libstdc++-5-dev` so I would recommend copying https://github.com/mapbox/mason/blob/master/scripts/cmake/3.8.2/.travis.yml to replace this file before you trigger builds.